components/script/dom/bindings/inheritance.rs

```diff
@@ -46,7 +46,7 @@ pub trait Castable: IDLInterface + DomObject + Sized {
         T: DerivedFrom<Self>,
     {
         if self.is::<T>() {
-            Some(unsafe { mem::transmute(self) })
+            Some(unsafe { mem::transmute::<&Self, &T>(self) })
         } else {
             None
         }
```

Comment:
What warning is fixed here?

Reply:
It fixes the [missing_transmute_annotations](https://rust-lang.github.io/rust-clippy/master/index.html#/missing_transmute_annotations) warning